maintain_pim/common_issues/index.rst

```diff
@@ -151,3 +151,23 @@ This parameter is set by the PIM at the index creation. If you want to apply it
                   }
               }
           }'
+    curl -XPUT 'localhost:9200/akeneo_pim_product_model/_settings' -H 'Content-Type: application/json' -d'
+          {
+              "index" : {
+                  "mapping" : {
+                      "total_fields" : {
+                          "limit" : "12000"
+                      }
+                  }
+              }
+          }'
+    curl -XPUT 'localhost:9200/akeneo_pim_product_and_product_model/_settings' -H 'Content-Type: application/json' -d'
+          {
+              "index" : {
+                  "mapping" : {
+                      "total_fields" : {
+                          "limit" : "12000"
+                      }
+                  }
+              }
+          }'
```

Comment:
You also have `product_proposal` for the proposal

Reply:
completed